gcloud-java-dns/src/main/java/com/google/gcloud/dns/ZoneInfo.java

```diff
@@ -18,6 +18,8 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
+import com.google.api.services.dns.model.ManagedZone;
+import com.google.common.base.Function;
```

Comment:
Are the two above imports ever used?

You use `ManagedZone` but it's fully qualified. This is issue is also present in `DnsImpl` where you import `com.google.api.services.dns.model.ManagedZone` and also use it as fully qualified in your code (e.g. in `listZones`).